packages/frontmatter/package.json

```diff
@@ -5,15 +5,18 @@
   "description": "Frontmatter types and validation for the mystjs ecosystem",
   "author": "Franklin Koch <franklin@curvenote.com>",
   "homepage": "https://github.com/curvenote/curvenote#readme",
-  "main": "./src/index.ts",
-  "types": "./src/index.ts",
-  "directories": {
-    "src": "src"
-  },
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
```

Comment:
@fwkoch, these changes stumped me for a while. Jest was breaking on this, just needed to export the built js, not the ts.